### DISCUSSION TITLE
"toRefs" is imported from external module "vue" but never used in xxx

### DISCUSSION THREAD

**Discussion by zeMinng** [vuejs/vitepress#4240] [General]
![image](https://github.com/user-attachments/assets/6415678e-5cc0-47e4-b1cf-dc290928fe65)
As shown in the picture, this warning occurs in build. Can it be avoided? I am not using it in my project and it should be vitepress something warned

**Comment by brc-dd** [comment]
Well, vitepress isn't using it either. Most likely coming from some of your other packages.

**Reply by zeMinng** [reply]
Thanks for your help, I will check out the other questions
